### DISCUSSION TITLE
Sehr coole Erweiterung

### DISCUSSION THREAD

**Discussion by Tastaturberuf** [mathContao/be_piwikcharts#6] [General]
Auch wieder eine der versteckten Perlen, die ich zufällig entdeckt habe. 

🧡

**Comment by math-GH** [comment]
Vielen Dank für das Lob.

Wie hast du die Erweiterung gefunden?

**Comment by Tastaturberuf** [comment]
Ich hab `matomo` im Repository eingeben, weil ich die Erweiterung von @chopsol gesucht habe. Ihr könntet euch auch gegenseitig suggesten. chopsol/contao-matomo-tracker

**Reply by math-GH** [reply]
Vielen Dank für die Empfehlung :)
